x-pack/plugins/security_solution/public/management/pages/policy/store/policy_list/test_mock_utils.ts

```diff
@@ -43,26 +44,28 @@ export const apiPathMockResponseProviders = {
             type: 'epm-packages',
             id: 'endpoint',
             attributes: {
-              installed: [
+              installed_kibana: [
```

Comment:
@paul-tavares are these changes okay?

Reply:
Yeah, this is all good with us. We don't really use it, but had to define it in the mock in order to make TS happy 😄